ping.py

```diff
@@ -63,19 +65,121 @@ def get_ident(): return 0
 MAX_SLEEP = 1000
 
 
-class MStats:
-    thisIP = "0.0.0.0"
-    pktsSent = 0
-    pktsRcvd = 0
-    minTime = 999999999
-    maxTime = 0
-    totTime = 0
-    avrgTime = 0
-    fracLoss = 1.0
+# noinspection PyAttributeOutsideInit
```

Comment:
Yeah, you're right...

I think this is a bad habit of mine... in projects with a lot of classes, I disable the inspection warnings on a class-by-class basis. And I forgot that this project has only a single class 😅